README.md

```diff
@@ -81,13 +76,13 @@ Please note that this package is still under active development and the API is s
 
 ## Config Options
 
-| Name                | Type     | Description                                                                   | Options                              | Required | Default |
-| ------------------- | -------- | ----------------------------------------------------------------------------- | ------------------------------------ | -------- | ------- |
-| `accessToken`       | `string` | API [OAuth token](https://developers.arcadia.com/#operation/createOAuthToken) |                                      | Yes      | none    |
-| `env`               | `string` | API environment                                                               | `['local', 'sandbox', 'production']` | Yes      | none    |
-| `newCredentialData` | `object` | Data used to prefill new credentials                                          | Content depends on value of `scope`  | No       | none    |
-| `callbacks`         | `object` | Callback functions                                                            |                                      | No       | none    |
-| `uiTheme`           | `string` | UI color theme                                                                | `['light', 'dark']`                  | No       | 'light' |
+| Name                  | Type     | Description                                                                                            | Options                              | Required | Default |
+| --------------------- | -------- | ------------------------------------------------------------------------------------------------------ | ------------------------------------ | -------- | ------- |
+| `utilityConnectToken` | `string` | [Utility connect access token](https://developers.arcadia.com/#section/Authentication/Utility-Connect) |                                      | Yes      | none    |
```

Comment:
I think we should standardize references to the Utility Connect Token to `UtilityConnectToken`.

Reply:
This is the param - it should be camel case here to match how you're passing it in.

Reply:
Sorry, my comment is about the reference in the Description column. My understanding is that the column is used to provide more context, not to demonstrate the exact parameter name